backend/src/main/java/com/project/G1_T3/player/controller/PlayerProfileController.java

```diff
@@ -23,7 +23,7 @@ public class PlayerProfileController {
     @PostMapping("/create") // Map to URL for creating player profile
     @ResponseStatus(HttpStatus.CREATED)
     public PlayerProfile create(@RequestBody PlayerProfile playerProfile) {
```

Comment:
Provide a more meaningful method name and don't use verbs as the mapping unless truly necessary